doc/dev_guide/testing.rst

```diff
@@ -59,32 +59,71 @@ First ensure pytest and its plugins are installed by:
 .. code:: bash
 
    # If using a standard hyperspy install
-   pip install hyperspy[tests]
+   $ pip install hyperspy[tests]
 
    # Or, from a hyperspy local development directory
-   pip install -e .[tests]
+   $ pip install -e .[tests]
 
    # Or just installing the dependencies using conda
-   conda install -c conda-forge pytest pytest-mpl
+   $ conda install -c conda-forge pytest pytest-mpl
 
 To run them:
 
 .. code:: bash
 
-   pytest --mpl --pyargs hyperspy
+   $ pytest --mpl --pyargs hyperspy
 
 Or, from HyperSpy's project folder, simply:
 
 .. code:: bash
 
-   pytest
+   $ pytest
 
 .. note::
 
   pytest configuration options are set in the ``setup.cfg`` file, under the
   ``[tool:pytest]`` section. See the `pytest configuration documentation
   <https://docs.pytest.org/en/latest/customize.html>`_ for more details.
 
+The HyperSpy test suite can also be run in parallel if you have multiple CPUs
+available, using the ```pytest-xdist`` plugin <https://pypi.org/project/pytest-xdist/>`_.
+If you have the plugin installed, HyperSpy will automatically run the test suite in
+parallel on your machine.
+
+.. code:: bash
+
+   # To run on all the cores of your machine
+   $ pytest -n auto --dist loadfile
+
+   # To run on 2 cores
+   $ pytest -n 2 --dist loadfile
+
+The ``--dist loadfile`` argument will group tests by their containing file. The 
+groups are then distributed to available workers as whole units, thus guaranteeing 
+that all tests in a file run in the same worker. 
+
```

Comment:
Advice to install `pytest-sugar` to make the results to the test suite more readable when using `pytest-xdist`?

It is not convenient to add to the `tests` extra requirement because we use it to install on travis...

Reply:
Sure!